### PR TITLE
Clustering Visualization

### PR DESCRIPTION
**Pull request by pnklshh**
1. Made visualization for clustering so that sentences in each cluster can be viewed separately.
2. Made number of clusters configurable - user can select it from the UI

### CODE REVIEW

File: backend.py [hunk 1]
```diff
@@ -28,17 +29,18 @@ def homepage():
         # ignore first and last characters i.e. '[' and ']' to get the list of line ids as a string like "1,3,6,7,...""
         if word_selected in dictionary:
             line_ids= str(dictionary[word_selected])[1:][:-1]
+            sentence_List = logic.SQLQuery(f"select Line_Text from {language_selected}_corpus where Line_id in ({line_ids})")
             clusteramount = request.form["clusteramount"]
             clusterlist = list(range(1,int(clusteramount)+1))
             sentence_List_clustered = kmeans.KMeansClustering(int(clusteramount),line_ids,language_selected)
-            #sentence_List = logic.SQLQuery(f"select Line_Text from {language_selected}_corpus where Line_id in ({line_ids})")
+        print(len(sentence_List))
```
Comment (oldosamaibraheem11) [inline backend.py]: I am assuming this was done for testing so we can remove this

File: templates/index.html [hunk 2]
```diff
@@ -38,28 +38,41 @@ <h1 id = "appname">Deelingual</h1>
...
-					{% for item in sentence_List %}
-					<li>{{item}}</li>
-					{% endfor %}
-				  </ol>
+			<h1 id = "resultslable"scope="col">{{word_translated_list[7]}}:</h1>
+			<p id = "error" class="error"><strong>{{ error }}</strong>
 		</div>
+		{% endif %}
+
+		{% for cluster in sentence_List_clustered %}
```
Comment (oldosamaibraheem11) [inline templates/index.html:57]: Very good, this makes the clusters very easy to read

**Review by oldosamaibraheem11 — CHANGES_REQUESTED** [review]
We just need to remove that print statement, everything looks good!

**Comment by pnklshh** [comment]
Made the change.

**Pull request merged**